README.md

```diff
@@ -17,310 +17,213 @@ Katib supports
-Set `MANIFESTS_DIR` to the cloned folder.
-export MANIFESTS_DIR=<cloned-folder>
-```
-
-### TF operator
-
-For installing TF operator, run the following:
-
-```
-cd "${MANIFESTS_DIR}/tf-training/tf-job-crds/base"
-kustomize build . | kubectl apply -f -
-cd "${MANIFESTS_DIR}/tf-training/tf-job-operator/base"
-kustomize build . | kubectl apply -f -
-```
-
-### PyTorch operator
-
-For installing PyTorch operator, run the following:
-
-```
-cd "${MANIFESTS_DIR}/pytorch-job/pytorch-job-crds/base"
-kustomize build . | kubectl apply -f -
-cd "${MANIFESTS_DIR}/pytorch-job/pytorch-operator/base/"
-kustomize build . | kubectl apply -f -
-```
-
-### Katib
-
-Note that your [kustomize](https://kustomize.io/) version should be >= 3.2.
-To install Katib run:
-
-```
-git clone git@github.com:kubeflow/katib.git
-make deploy
-```
-
-Check if all components are running successfully:
-
-```
-kubectl get pods -n kubeflow
-```
-
-Expected output:
-
-```
-NAME                                READY   STATUS    RESTARTS   AGE
-katib-controller-858d6cc48c-df9jc   1/1     Running   1          20m
-katib-db-manager-7966fbdf9b-w2tn8   1/1     Running   0          20m
-katib-mysql-7f8bc6956f-898f9        1/1     Running   0          20m
-katib-ui-7cf9f967bf-nm72p           1/1     Running   0          20m
-pytorch-operator-55f966b548-9gq9v   1/1     Running   0          20m
-tf-job-operator-796b4747d8-4fh82    1/1     Running   0          21m
-```
-
-### Running examples
+# Documentation
 
-After deploy everything, you can run examples to verify the installation.
+- Run your first Katib Experiment in the
+  [getting started guide](https://www.kubeflow.org/docs/components/katib/hyperparameter/#example-using-random-algorithm).
 
-This is an example for TF operator:
+- Learn about Katib concepts in the
+  [Kubeflow documentation](https://www.kubeflow.org/docs/components/katib/overview/#katib-concepts).
 
-```
-kubectl create -f https://raw.githubusercontent.com/kubeflow/katib/master/examples/v1beta1/tfjob-example.yaml
-```
-
-This is an example for PyTorch operator:
-
-```
-kubectl create -f https://raw.githubusercontent.com/kubeflow/katib/master/examples/v1beta1/pytorchjob-example.yaml
-```
-
-Check the
-[Kubeflow documentation](https://www.kubeflow.org/docs/components/katib/hyperparameter/#example-using-random-algorithm)
-how to monitor your `Experiment` status.
-
-You can view your results in Katib UI.
-If you used standard installation, access the Katib UI via Kubeflow dashboard.
-Otherwise, port-forward the `katib-ui`:
-
-```
-kubectl -n kubeflow port-forward svc/katib-ui 8080:80
-```
-
-You can access the Katib UI using this URL: `http://localhost:8080/katib/`.
-
-### Katib SDK
+- Know more about various Katib interfaces in the
+  [introduction guide](https://www.kubeflow.org/docs/components/katib/overview/#katib-interfaces).
 
-Katib supports Python SDK:
+- TODO (andreyvelich): Add Katib components section from the website.
```

Comment:
Should we remove this till we add this section?

Reply:
Sure, does it look good to you: https://github.com/kubeflow/katib/blob/e2434e3c3c4001280d8ca1eb9d0af74fd17aed75/README.md#documentation ?
I made the similar sentences.